src/test/java/com/github/dockerjava/core/util/CertificateUtilsTest.java

```diff
@@ -0,0 +1,28 @@
+package com.github.dockerjava.core.util;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+public class CertificateUtilsTest {
```

Comment:
Classloader will exist because it runs under test classloader and resources also will be under path because tests run under this dir. No need to do file creations, just use static files but from package based path (atm moment resources folder is trashed and it unclear to what tests belongs files).